packages/atomic/src/components/commerce/product-template-components/atomic-product-children/atomic-product-children.pcss

```diff
@@ -0,0 +1,7 @@
+@import '../../../../global/global.pcss';
+
+.active-child {
```

Comment:
Could set border radius and box sizing directly in `tsx` code as taildwind classes : `box-border rounded`